### ISSUE TITLE
Test failure

### ISSUE THREAD

**Issue by Fale** [innotop/innotop#117]
I'm building innotop (last master version) for Fedora, and I'm seeing some test failing:

``` bash
Executing(%check): /bin/sh -e /var/tmp/rpm-tmp.Ewk4UN
+ umask 022
+ cd /builddir/build/BUILD
+ cd innotop-4131ed53b5bebbfdea79a9b090692a246063a419
+ make test
PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
Use of uninitialized value $mysqlversion in pattern match (m//) at /builddir/build/BUILD/innotop-4131ed53b5bebbfdea79a9b090692a246063a419/innotop line 474.
# Looks like you planned 5 tests but ran 1.
# Looks like your test exited with 255 just after 1.
t/InnoDBParser.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 4/5 subtests 
Test Summary Report
-------------------
t/InnoDBParser.t (Wstat: 65280 Tests: 1 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 5 tests but ran 1.
Files=1, Tests=1,  1 wallclock secs ( 0.08 usr  0.00 sys +  0.71 cusr  0.06 csys =  0.85 CPU)
Result: FAIL
Failed 1/1 test programs. 0/1 subtests failed.
```

I think the cause is the version of Test::Harness. Recent version have changes with the behaviour of Test::Harness::Switches

**Comment by chenzhe07** [comment]
```
--- /tmp/innotop-1.10.0/t/InnoDBParser.t	2015-12-08 10:52:44.585802160 +0800
+++ InnoDBParser.t	2015-12-08 11:33:33.988691683 +0800
@@ -33,7 +33,7 @@
    $path . 't/innodb-status-006' => "5.5or-earlier",
    $path . 't/innodb-status-007' => "5.5or-earlier",
    $path . 't/innodb-status-008' => "5.6.12",
-   $path . 't/innodb-status-008' => "5.7"
+   $path . 't/innodb-status-009' => "5.7"
 );
 
 my %tests = (
@@ -2186,6 +2186,7 @@
 
 foreach my $fn ( keys %tests ) {
    my $contents = read_file($fn);
+   print "$fn" . "\n";
    my %result   = $innodb_parser->get_status_hash($contents, undef, undef, undef, $versions{$fn});
    local $Data::Dumper::Indent = 2;
    #print Dumper \%result;
```
the hash key in %version and %tests  was not matched in InnoDBParser.t, you can change it and test it again.

**Comment by Fale** [comment]
it seems like others tests fails as well :(

**Comment by yoku0825** [comment]
I pull-requested to fix by #135

**Comment by yoku0825** [comment]
This problem has been fixed. Close?
